lib/bolt/cli.rb

```diff
@@ -353,5 +357,16 @@ def file_stat(path)
     def outputter
       @outputter ||= Bolt::Outputter.for_format(config[:format], config[:color], config[:trace])
     end
+
+    def load_bundled_content
+      default_content = Bolt::PAL.new(Bolt::Config.new)
```

Comment:
It looks like this should avoid picking up the default modulepath and just use the vendored modules. It also doesn't appear to have any adverse impact on performance. So this approach seems ok.

I'd rename this to `bundled_content` and just have it called directly when constructing the Executor. No need to have separate checks beforehand.

Reply:
Oh right, task/plan are mutually exclusive on CLI. Will clean that up.